### DISCUSSION TITLE
question about COD and .net

### DISCUSSION THREAD

**Discussion by runup-yt** [dotnet/runtime#79456] [Q&A]
do i need this to play call of duty black ops 1 if i do what version and also do i need .net or .net framework or runtime because i'm keep on getting the error 0xc000007b and i checked the computer and the computer for directx and it has it. It has directx 12

**Comment by Clockwork-Muse** [comment]
You do not need the direct contents of this repository.
You may need one of the distributed runtime packages (more likely one of the .NET Framework ones), however Windows normally automatically installs or prompts you for install.  If it doesn't [you can download .NET Framerwork packages manually](https://dotnet.microsoft.com/en-us/download/dotnet-framework), based on the one it claims that you're missing.
